cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1053,6 +1053,21 @@
       public static final String INTERNAL_CERT_PASSWORD = "ssl.internal.cert.password";
     }
 
+    /**
+     * Authentication.
+     */
+    public static final class Authentication {
+      /**
+       * Determines which authentication mode to use.
+       * Should be chosen from the {@link io.cdap.cdap.security.auth.AuthenticationMode} enum.
+       */
+      public static final String AUTHENTICATION_MODE = "security.authentication.mode";
```

Comment:
The mode and the header configs should also be in the cdap-default.xml. Typically we have all configurations in the cdap-default.xml to have it serving as a central place to see all available configurations.

Reply:
Done, thanks for review =)